### PR TITLE
WS-02_boilerplate_pushen

### LINKED ISSUE #2: Boilerplate code pushen
- vite
- react/typescript
- react-three-fiber
- react-thre-drei
- Canvas Scene

### PR DESCRIPTION
**Pull request by TaoN23**
- setup ✅
- dependencies installed (r3f, r3drei, types: three) ✅
- basic Scene setup ✅

closes #2

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  button {
+    background-color: #f9f9f9;
+  }
+}
+
+*{
+  width: 100vw;
+  height: 100vh;
+  margin: 0px;
+}
```
Comment (13Anonym) [inline src/index.css:76]: A line at the end is missing ;)

Comment (TaoN23) [reply src/index.css]: new line added ✅

File: src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { Box, Sky } from "@react-three/drei"
+import { Canvas } from "@react-three/fiber"
+
+function App() {
+
```
Comment (13Anonym) [inline src/App.tsx:5]: Maybe you can start to comment also with easy lines of code for a better overview ;)

Comment (TaoN23) [reply src/App.tsx:5]: Next time :)

**Review by 13Anonym — COMMENTED** [review]
Looks good ;)

**Pull request merged**